tests/Fixer/Import/FullyQualifiedStrictTypesFixerTest.php

```diff
@@ -376,6 +410,8 @@ function A(\Z\B\C\DE\Foo $fix) {}
 namespace Foo\Test;
 use Other\BaseClass;
 use Other\CaughtThrowable;
+use Other\Cl;
+use Other\Cl2;
```

Comment:
Please use meaningful names like in the other symbol in this test case, like:


```suggestion
use Other\InstantiatedClass;
use Other\InstanceOfClass;
```

It does not matter much, but would be consistent.

Reply:
👍 done